frontend/src/components/bauvorhaben/BauvorhabenForm.vue

```diff
@@ -215,6 +215,12 @@
         </v-col>
       </v-row>
     </field-group-card>
+    <field-group-card
+      v-if="!isNew"
+      :card-title="referencedObjectsCardTitle"
+    >
+      <referenced-items-list />
```

Comment:
Weiteres Review abhängig von Review https://github.com/it-at-m/isi-backend/pull/67#discussion_r1274724959